src/ansys/mapdl/core/launcher.py

```diff
@@ -1750,11 +1750,25 @@ def launch_mapdl(
         check_valid_port(port)
         LOG.debug(f"Using default port {port}")
 
+    # verify version
+    if exec_file and version:
+        raise ValueError("Cannot specify both ``exec_file`` and ``version``.")
+
+    if version is None:
+        version = os.getenv("PYMAPDL_MAPDL_VERSION", None)
+
+    version = _verify_version(version)  # return a int version or none
```

Comment:
It looks like it's always the three digits version (like `232`), but please double check if possible, it's the format expected by PIM (see https://fuji.ansysapis.com/docs/status/catalog/platform_services/pim/#product-naming)

Reply:
Double checked.